### DISCUSSION TITLE
Question about parsing

### DISCUSSION THREAD

**Discussion by mugeunson** [CederGroupHub/chgnet#175] [Q&A] [answered]
Hi, first of all, thanks for chgnet developer.

Currently, i am performing fine-tune with vasp data.
In this process, I was wondering, some vasp data can be fine-tuned, but other vasp data results in m_MAE key errors.

Maybe, I think this difference is derived magnetic moment.
If an error occurred, i gave ispin=2 but did not set LORBIT set to true; if no error occurred, i set LORBIT set to true. 
But, as far as I know, setting ispin=2 will set the magnetic moment to the default value, but why does fine-tuning in this case cause m_MAE error?

Could you please tell me why this is happens?

thank you!

**Comment by janosh** [comment]
Please post the full stack trace and an example fine-tuning vasp run dir

**Reply by mugeunson** [reply]
If I don't set LORBIT to true in the INCAR file in the vasp data I am training CHGNet on, I get an m_MAE key error.
sorry, I am not sure what the requested FULL STACK TRACE is, so I have attached a picture of the error that occurred.

My final concern is whether the data is inappropriate for training CHGNet if I do not set LORBIT=true.

I would appreciate it if you could tell me what more is needed to resolve this question.

thank you!
![m_MAE_key_error](https://github.com/CederGroupHub/chgnet/assets/139963735/7d75820b-3803-49e6-9947-27a70d3aaad2)

**Reply by janosh** [reply]
when debugging, it helps be able to reproduce your error. for that, we need a minimal example, i.e. the smallest piece of code and data that triggers the error. the stack trace you posted is helpful but it's better to post it as text rather than an image to make it searchable.

**Reply by mugeunson** [reply]
Thank you for your kind response. As you requested, I have attached the LORBIT_FALSE folder with errors and the LORBIT_TRUE folder without errors along with the code.

[example.zip](https://github.com/user-attachments/files/16108954/example.zip)

**Reply by janosh** [reply, marked as answer]
the model trains fine for me on `LORBIT_FALSE ` if i drop the `m` from `targets="efsm"`:
```diff
trainer = Trainer(
    model=chgnet,
-   targets="efsm",
+   targets="efs",
    optimizer="Adam",
    scheduler="CosLR",
    criterion="MSE",
    epochs=50,
    learning_rate=1e-3,
    use_device="cpu",
    print_freq=6,
)
trainer.train(train_loader, val_loader, test_loader)
```

**Reply by mugeunson** [reply]
Thanks to you, the problem is solved. Thank you very much!
